TicTacToeApp/ViewController.swift

```diff
@@ -8,18 +8,38 @@
 
 import UIKit
 
-class ViewController: UIViewController {
+public class ViewController: UIViewController {
+  var currentPlayerMark = "X"
```

Comment:
Yeah, that variable doesn't exist in ViewController as it is now. ViewController delegates to Game to track the current player. Game does this with a private variable called currentPlayer.